src/lib/api/attachment.ts

```diff
@@ -13,6 +13,7 @@ export async function getAttachmentUrl(attachment: { key: string }): Promise<str
     .query({
       key: attachment.key,
     })
+    .set('X-APP-PLATFORM', 'zos')
```

Comment:
Could we change this method (`getAttachmentUrl`) and this file to use the api/rest.ts methods instead of manually setting the header? Then we don't have to manually set the header.

Reply:
Good idea.